README.md

```diff
@@ -637,56 +641,57 @@ The Winkel tripel projection.
 
 ### Raw Projections
 
-Raw projections are typically passed to the [d3.geoProjection](#geoProjection) constructor. They are exposed here to facilitate the derivation of related projections.
+Raw projections are typically passed to the [d3.geoProjection](#geoProjection) constructor. They are exposed here to facilitate the derivation of related projections. In raw projections all angles are expressed in radians.
```

Comment:
I’m tempted to leave this last sentence out. We already say “in radians” below for lambda and phi, and there are no restrictions on (or explanation of) the parameters for each raw projection. Like for the bottomley projection you can sort of think of the ψ parameter as an angle, but it’s expressed as sin(ψ) not as radians… It’d be better to document the parameters individually than make a general statement here.